CHANGELOG.md

```diff
@@ -3,6 +3,7 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 ### Added
+- OxyPlot.Series.HeatMapSeries.GetValueForLabels (#256)
```

Comment:
GetValueForLabels is not a public member - and no functionality added? then it should not be necessary to add it here